obal/data/roles/build_package/tasks/koji_build.yml

```diff
@@ -1,50 +1,76 @@
 ---
 - name: "Package name"
-  rpm_nevr:
+  rpm_nvr:
     spec_file: "{{ spec_file_path }}"
     scl: "{{ tag.scl | default(omit) }}"
     dist: "{{ tag.dist | default(omit) }}"
     macros: "{{ tag.macros | default(omit) }}"
-  register: package_nevr
-
-- name: create temporary build directory
-  tempfile:
-    state: directory
-    suffix: srpms
-  register: srpm_directory
-  when: srpm_directory is not defined
-
-- name: 'Build SRPM'
-  srpm:
-    package: "{{ inventory_dir }}/{{ package_base_dir }}/{{ inventory_hostname }}"
-    output: "{{ srpm_directory.path if 'path' in srpm_directory else srpm_directory }}"
-    scl: "{{ scl | default(omit) }}"
-    source_location: "{{ source_location | default(omit) }}"
-    source_system: "{{ source_system | default(omit) }}"
-  register: srpm_build
-
-- name: "Build package in Koji"
-  koji_build:
-    scratch: "{{ build_package_scratch }}"
+  register: package_nvr
+
+- name: Check build status
+  check_koji_build:
     tag: "{{ tag.name }}"
-    srpm: "{{ srpm_build.path }}"
-    package: "{{ package_nevr.name }}"
-    nevr: "{{ package_nevr.nevr }}"
+    nvr: "{{ package_nvr.nvr }}"
+    package: "{{ package_nvr.name }}"
     koji_executable: "{{ build_package_koji_command }}"
-    tag_check: "{{ build_package_koji_whitelist_check }}"
-  register: koji_build_tasks
-
-- name: 'Created tasks'
-  debug:
-    var: koji_build_tasks.task_urls
-  when: koji_build_tasks.changed
-
-- name: 'Wait for tasks to finish'
-  include_tasks: wait.yml
-  when: build_package_wait|bool and koji_build_tasks.changed
-  vars:
-    koji_tasks: "{{ koji_build_tasks.tasks }}"
-
-- name: 'Download task results'
-  include_tasks: download_rpms.yml
-  when: build_package_download_rpms|bool
+  register: build_exists
+  when: not build_package_scratch
+
+- debug:
+    msg: "{{ build_exists }}"
```

Comment:
should this be a nicer message (or removed?)

Reply:
this is the debug output of `diff_package` from https://github.com/theforeman/obal/blob/6f66c0caac0409488312a8062ef77fc4e09138d3/obal/data/roles/diff_package/tasks/koji_old.yml#L31-L36

(well, or your edited version of that)

but the comment here is on he `build_package` role and the variable is coming from the `register` step before.

Reply:
Oh duh, I forgot to pass the flag when generating the output:

```
TASK [build_package : debug] *************************************************************************************************************************************************************************************************************************************************************
ok: [rubygem-deep_cloneable] => 
  msg:
    changed: false
    exists: true
    exists_for_tag: true
    failed: false
    tagged_version: rubygem-deep_cloneable-3.0.0-4.el8
```

Reply:
nah, works for me!